00-RELEASENOTES

```diff
@@ -1,16 +1,146 @@
+* Introduce bgsave cancel (#757)
+* Add paused_reason to INFO CLIENTS (#1564)
+* show capa in CLIENT LIST / CLIENT INFO (#1698)
+* Set Command with IFEQ Support (#1324)
+* Add new flag in `CLIENT LIST` for import-source client (#1398)
+* Adding Missing filters to CLIENT LIST and Dedup Parsing (#1401)
+* Add `availability_zone` to the HELLO response (#1487)
+* Import-mode: Avoid expiration and eviction during data syncing (#1185)
 
-More information is available at https://valkey.io
+Observability and Monitoring changes
+====================================
+* Extend LATENCY LATEST to add sum / cnt stats (#1570)
+* add paused_actions for INFO Clients (#1519)
+* Feature COMMANDLOG to record slow execution and large request/reply (#1294)
+* Fix cluster info sent stats for message with light header (#1563)
+* Add latency stats around cluster config file operations (#1534)
 
-Happy hacking!
+Performance/Efficiency Improvements - Core
+==========================================
+* New hash table (#1186)
+* Free strings during BGSAVE/BGAOFRW to reduce copy-on-write (#905)
+* Creates an empty lua table with specified initial capacity as much as possible (#1092)
+* Move prepareClientToWrite out of loop for HGETALL command (#1119)
+* Improved hashing algorithm in luaS_newlstr (#1168)
+* Replace dict with new hashtable for sets datatype (#1176)
+* Do security attack check only when command not found to reduce the critical path. (#1212)
+* Accelerate hash table iterator with prefetching (#1501)
+* Replace dict with new hashtable: hash datatype (#1502)
+* Replace dict with new hashtable for sets datatype (#1176)
+* Replace dict with new hashtable: sorted set datatype (#1427)
+* Trim free space from inline command argument strings to avoid excess memory usage (#1213)
+* Increase the IO_THREADS_MAX_NUM. (#1220)
+* Refactor of ActiveDefrag to reduce latencies (#1242)
+* Integrating fast_float to optionally replace strtod (#1260)
+* Improvements for TLS with I/O threads (#1271)
+* Optimize PFCOUNT, PFMERGE command by SIMD acceleration (#1293)
+* Optimize sdscatrepr by batch processing printable characters (#1342)
+* Optimize ZRANK to avoid path comparisons (#1389)
+* move clientCron onto a separate timer (#1387)
+* client struct: lazy init components and optimize struct layout (#1405)
+* Offload reading the replication stream to IO threads (#1449)
+* skip CRC checksumming during diskless full sync with TLS enabled. (#1479)
+* Accelerate hash table iterator with value prefetching (#1568)
+
+New/Modified configurations
+===========================
+* Add io-threads-do-reads config to deprecated config table to have no effect. (#1138)
+
+Build and Packaging changes
+=================================
+* RDMA builtin support (#1209)
+* Fix Valkey binary build workflow, version support changes. (#1429)
+
+Code improvements
+=================
+* Remove valkey specific changes in jemalloc source code (#1266)
+
+Module Improvements
+===================
```

Comment:
Please add PR https://github.com/valkey-io/valkey/pull/1041. It is related to module part, Thanks

Reply:
this was the most complicated piece of work I have ever did in Valkey - @hwware can you take a look if that seems right?